common/src/main/java/dev/latvian/mods/kubejs/create/KubeJSCreatePlugin.java

```diff
@@ -1,29 +1,51 @@
 package dev.latvian.mods.kubejs.create;
 
+import com.google.common.collect.ImmutableMap;
 import com.simibubi.create.AllRecipeTypes;
 import com.simibubi.create.content.contraptions.processing.ProcessingRecipeSerializer;
 import dev.latvian.mods.kubejs.KubeJSPlugin;
 import dev.latvian.mods.kubejs.RegistryObjectBuilderTypes;
+import dev.latvian.mods.kubejs.create.events.BoilerHeaterHandlerEvent;
+import dev.latvian.mods.kubejs.create.events.SpecialFluidHandlerEvent;
+import dev.latvian.mods.kubejs.create.events.SpecialSpoutHandlerEvent;
+import dev.latvian.mods.kubejs.recipe.RecipeJS;
 import dev.latvian.mods.kubejs.recipe.RegisterRecipeHandlersEvent;
+import dev.latvian.mods.kubejs.script.ScriptType;
 import net.minecraft.resources.ResourceLocation;
 
+import java.util.Map;
+import java.util.function.Supplier;
+
 /**
  * @author LatvianModder
  */
 public class KubeJSCreatePlugin extends KubeJSPlugin {
+
+	private static final Map<ResourceLocation, Supplier<RecipeJS>> recipeProviders = ImmutableMap.<ResourceLocation, Supplier<RecipeJS>>builder()
+			.put(new ResourceLocation("create:deploying"), ItemApplicationRecipeJS::new)
+			.put(new ResourceLocation("create:item_application"), ItemApplicationRecipeJS::new)
+			.build();
```

Comment:
_For the sake of code style consistency:_ Please use `Map.of()` instead of Google's ImmutableMap

Reply:
Done.